libp2p/exceptions.py

```diff
@@ -13,4 +18,11 @@ class ParseError(BaseLibp2pError):
 class MultiError(BaseLibp2pError):
     """Raised with multiple exceptions."""
```

Comment:
Please expand on the docstring, at least a better description of when it's raised and maybe an example of what the output should look like.